keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -0,0 +1,90 @@
+from keras import backend
+from keras.engine import base_layer
+from keras.engine import base_preprocessing_layer
+from keras.layers.preprocessing import preprocessing_utils as utils
+from keras.utils import image_utils
+from keras.utils import tf_utils
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.util.tf_export import keras_export
+from tensorflow.tools.docs import doc_controls
+
+H_AXIS = -3
+W_AXIS = -2
+
+
+class RandomResizedCrop(base_layer.Layer):
+    '''
+    This layer is used to
+    Crop a random portion of image and resize it to a given size.
+    A crop of the original image is made: the crop has a random area (H * W) and a random aspect ratio.
+    This crop is finally resized to the given size.
+    This is popularly used to train the Inception networks.
+    '''
+
+    def __init__(self, height, width, seed=None, **kwargs):
+        base_preprocessing_layer.keras_kpl_gauge.get_cell('RandomCrop').set(True)
+        super(RandomResizedCrop, self).__init__(**kwargs, autocast=False, seed=seed,
```

Comment:
in python3 you can just call `super().__init__`

Reply:
Done : Removed ``` super(RandomResizedCrop, self).__init__(**kwargs, autocast=False, seed=seed,```
Added : ``` super().__init__```